plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -861,11 +861,11 @@ function MosaicMenu:_recalculateDimen()
     local portrait_mode = Screen:getWidth() <= Screen:getHeight()
     -- 3 x 3 grid by default if not initially provided (4 x 2 in landscape mode)
     if portrait_mode then
-        self.nb_cols = self.nb_cols_portrait or 3
-        self.nb_rows = self.nb_rows_portrait or 3
+        self.nb_cols = tonumber(BookInfoManager:getSetting("nb_cols_portrait") or 3)
```

Comment:
What's the reasoning behind using `tonumber` here?

Reply:
```Lua
print("TYPE", type(BookInfoManager:getSetting("nb_cols_portrait")))

TYPE	string
```

Reply:
https://github.com/koreader/koreader/blob/5a4e70a19da4e77bdd80f2335145004466fbce6b/plugins/coverbrowser.koplugin/bookinfomanager.lua#L366-L369

Reply:
Gah, thought this came from a LuaSettings, but, nope...

```
    -- To keep track of CoverBrowser settings
    CREATE TABLE IF NOT EXISTS config (
        key TEXT PRIMARY KEY,
        value TEXT
    );
```